src/main/java/groovy/lang/GroovyClassLoader.java

```diff
@@ -703,10 +703,10 @@ public Collection getLoadedClasses() {
     }
 
     /**
-     * open up the super class define that takes raw bytes
+     * Converts an array of bytes into an instance of {@code Class}.
      */
-    public Class defineClass(String name, byte[] b) {
```

Comment:
Is it normal to add an unchecked exception to a throws clause?

Reply:
It seems that the PR derives the exceptiion definition from the following method:

**java.lang.ClassLoader#defineClass(java.lang.String, byte[], int, int)**
```java
    protected final Class<?> defineClass(String name, byte[] b, int off, int len)
        throws ClassFormatError
    {
        return defineClass(name, b, off, len, null);
    }
```